nilearn/_utils/niimg.py

```diff
@@ -269,20 +282,22 @@ def _short_repr(niimg_rep, shorten=True, truncate=20):
 
 
 def img_data_dtype(niimg):
-    """Determine type of data contained in image
+    """Determine type of data contained in image.
 
     Based on the information contained in ``niimg.dataobj``, determine the
     dtype of ``np.array(niimg.dataobj).dtype``.
     """
-
     dataobj = niimg.dataobj
 
     # Neuroimages that scale data should be interpreted as floating point
-    if nibabel.is_proxy(dataobj) and (dataobj.slope, dataobj.inter) != (1.0, 0.0):
+    if nibabel.is_proxy(dataobj) and (dataobj.slope, dataobj.inter) != (
+        1.0,
+        0.0,
+    ):
```

Comment:
I think we can trick black by splitting this manually in a more human-readable way:

```python
if (
    nibabel.is_proxy(dataobj) and
    (dataobj.slope, dataobj.inter) != (1.0, 0.0)
):
    .....
```

Reply:
black is opinionated: it will always give me this formatting no matter what I try